internal/cmd/config.go

```diff
@@ -46,6 +46,8 @@ func runConfig(_ *cobra.Command, _ []string) error {
 	if err != nil {
 		return err
 	}
+
+	//nolint:forbidigo // This is an expected print statement.
 	fmt.Printf("Config sourced from: %s\n\n%s\n", config.Path, marshaled)
```

Comment:
One way to circumvent the linter is to use "fmt.Fprintf(os.StdOut, " 

fmt.Println are often left behind debug, so the forbidigo default value are good

Your solution of using nolint is good, but I wanted to share what else could be used

Reply:
Ah, that's interesting, I'll see if I prefer the `nolint` or the `Fprintf` method :)
Thanks for letting me know.